src/video/VideoList.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import VideoRow from './VideoRow.js'
+
+const VideoList = (videos) => (
+    <ul>
+        { videos.map((v) =>
```

Comment:
@borisyankov here gives me error where videos does not have map function, did I pass it wrongly?